harness/tests/cli/test_requests.py

```diff
@@ -0,0 +1,135 @@
+import dataclasses
+import json
+import os
+import tempfile
+import uuid
+from pathlib import Path
+from types import SimpleNamespace
+from typing import Optional
+
+import pytest
+import requests
+import requests_mock as mock
+
+import determined
+import determined.cli
+import determined.cli.cli as cli
+import determined.cli.command as command
+from determined import common
+from determined.common import constants, context
+from determined.common.api import Session, bindings, certs
+from determined.common.api.authentication import Authentication
+from tests.common import api_server
+from tests.filetree import FileTree
+
+
+@dataclasses.dataclass
+class CliArgs:
+    master: str
+    experiment_id: int
+    user: str = "test"
+    password: str = "test"
+    polling_interval: int = 1
+
+
+@pytest.fixture
+def experiment_json():
+    with open(api_server.FIXTURES_DIR / "experiment.json") as f:
+        return json.load(f)["experiment"]
```

Comment:
why not just hardcode this json into python?

Reply:
hmm, my thinking is if we're not modifying it makes sense to keep it in a JSON filetype. this would be slower compared to what you're suggesting though